From 5c514f18041db9031564ec0bc7304f896a96eca6 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:40 -0400
Subject: [redhat] kabi: Adding symbol proc_dointvec (kernel/sysctl.c)

Message-id: <e7a86eae50e2d96b0257c317b9e0b40cac2617ef.1410510773.git.poros@redhat.com>
Patchwork-id: 96045
O-Subject: [PATCH RHEL7.1 160/233] kabi: Adding symbol proc_dointvec (kernel/sysctl.c)
Bugzilla: 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Read a vector of integers.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/proc_dointvec
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec b/redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec
new file mode 100644
index 0000000..a5e3f94
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec
@@ -0,0 +1,2 @@
+#1-
+0x1857aa39 proc_dointvec vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/proc_dointvec b/redhat/kabi/kabi-module/kabi_s390x/proc_dointvec
new file mode 100644
index 0000000..b60276e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/proc_dointvec
@@ -0,0 +1,2 @@
+#1-
+0xca03281c proc_dointvec vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec b/redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec
new file mode 100644
index 0000000..40f6336
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec
@@ -0,0 +1,2 @@
+#1-
+0xd067fc5c proc_dointvec vmlinux EXPORT_SYMBOL
-- 
1.7.1